Statistics/Histograms/Histogram.cs

```diff
@@ -799,6 +824,35 @@ public Int64 EstimateIterationsRemaining(double upperq, double lowerq)
             return Convert.ToInt64(Math.Min(remainingIters, biggestGuess));
         }
 
+        public string Print(bool round = false)
```

Comment:
I don't like this name. Print makes me think print to console. like ConsoleWriteLine. I'd maybe call this PropertiesAsStringMessage or PropertiesAsMessage - whatever better fits it's use case.

Reply:
trouble is - I am not sure what is the intended use - this is a remnant of our predecessor's code

Reply:
understood. I hate it, but it's not critical right now.